### PR TITLE
fix: Markdownlint cleanup for ES

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/es/web/html/element/input/checkbox/index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Este elemento posee los "[atributos globales](/es/docs/HTML/Global_attributes)".
 - {{htmlattrdef("checked")}}
   - : Cuando el valor del atributo **type** es **`checkbox`**, la presencia de este atributo booleano indica que el control se selecciona de forma predeterminada; de lo contrario, se ignora.
 - {{htmlattrdef("value")}}
-  - : El valor inicial de control. Si se omite la propiedad **value**, el resultado\*\* **devuelto al leer esta propiedad será la cadena **on.\*\*
+  - : El valor inicial de control. Si se omite la propiedad **value**, el resultado **devuelto al leer esta propiedad será la cadena **on.**
```
Comment (Graywolf9) [inline files/es/web/html/element/input/checkbox/index.md:16]:
```suggestion
  - : El valor inicial de control. Si se omite la propiedad **value**, el **resultado devuelto** al leer esta propiedad será la cadena **on**.
```

File: files/es/web/http/cors/index.md [hunk 2]
```diff
@@ -295,7 +295,9 @@ Content-Type: text/plain
...
+Pese a que la línea 11 contiene la Cookie destinada para el contenido en `http://bar.other`, si bar.other no responde con `Access-Control-Allow-Credentials: true` (línea 19) la respuesta será ignorada y no puesta a disposición para el contenido web.
+
+> **Nota Importante:** cuando se responde a una solicitud con credeciales, el servidor **debe** especificar un dominio, y no puede usar comodines. El ejemplo de arriba fallará si la cabecera fuera un comodín como: `Access-Control-Allow-Origin:`. Dado que `Access-Control-Allow-Origin` menciona explícitamente `http://foo.example`, el contenido de credenciales competente es devuelto al contenido web invocador. Observe que, en la línea 22, se establece una cookie adicional.
```
Comment (Graywolf9) [inline files/es/web/http/cors/index.md:300]:
```suggestion
> **Nota:** cuando se responde a una solicitud con credeciales, el servidor **debe** especificar un dominio, y no puede usar comodines. El ejemplo de arriba fallará si la cabecera fuera un comodín como: `Access-Control-Allow-Origin:`. Dado que `Access-Control-Allow-Origin` menciona explícitamente `http://foo.example`, el contenido de credenciales competente es devuelto al contenido web invocador. Observe que, en la línea 22, se establece una cookie adicional.
```

Comment (nschonni) [reply files/es/web/http/cors/index.md:300]: FYI, I'm not sure this Note exists in the en_US, but it seemed odd that it was embedded in the sentence so I split it

File: files/es/web/javascript/reference/index.md [hunk 3]
```diff
@@ -202,7 +202,7 @@ Esta parte de la sección de Javascript en MDN sirve como un repositorio de hech
 ### Expresiones del lado izquierdo
 
 - {{JSxRef("Operators/Property_accessors", "Property accessors", "", 1)}}
-- {{JSxRef("Operators/Optional_chaining", "<code>?.</code> (Optional chaining)", "", 1)}}
+- [`?.` (Optional chaining)](/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
```
Comment (Graywolf9) [inline files/es/web/javascript/reference/index.md:205]:
```suggestion
- [`?.` (Optional chaining)](/es/docs/Web/JavaScript/Reference/Operators/Optional_chaining)
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much!

**Pull request merged**